### ISSUE TITLE
Add Images form - Add button should have keyboard focus

### ISSUE THREAD

**Issue by kphunter** [bcit-tlu/hriv#662]
Currently, when adding a new image and updating the image title in the Name field, when the "Return" keyboard key is pressed, nothing happens. This is presumably because the mouse focus is still in the Name field.

The form should allow the user to finish typing and then press enter to save and submit the form.
